kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/appliers/artifacts/JvmVariantArtifacts.kt

```diff
@@ -48,9 +49,25 @@ internal class JvmVariantArtifacts(
         }
 
         fromOrigin(variantOrigin) { compilationName ->
-            compilationName !in variantConfig.sources.excludedSourceSets.get()
-                    && compilationName != "test"
+            compilationIsExcluded(compilationName, variantConfig.sources)
         }
     }
 
+    private fun compilationIsExcluded(compilationName: String, variant: KoverVariantSources): Boolean {
+        if (compilationName in variant.excludedSourceSets.get()) {
+            return false
+        }
+
+        val included = variant.includedSourceSets.get()
+
+        if (included.isEmpty() && compilationName == "test") {
```

Comment:
Are tests excluded by default? Is it mentioned somewhere in the documentation?

Reply:
Yes, `test` source set excluded by default.